src/python/pants/bin/remote_pants_runner.py

```diff
@@ -69,27 +69,21 @@ def __init__(
         args: List[str],
         env: Mapping[str, str],
         options_bootstrapper: OptionsBootstrapper,
-        stdin=None,
-        stdout=None,
-        stderr=None,
     ) -> None:
         """
         :param args: The arguments (e.g. sys.argv) for this run.
         :param env: The environment (e.g. os.environ) for this run.
         :param options_bootstrapper: The bootstrap options.
-        :param file stdin: The stream representing stdin.
-        :param file stdout: The stream representing stdout.
-        :param file stderr: The stream representing stderr.
         """
         self._start_time = time.time()
         self._args = args
         self._env = env
         self._options_bootstrapper = options_bootstrapper
         self._bootstrap_options = options_bootstrapper.bootstrap_options
         self._client = PantsDaemonClient(self._bootstrap_options)
-        self._stdin = stdin or sys.stdin
-        self._stdout = stdout or sys.stdout.buffer
-        self._stderr = stderr or sys.stderr.buffer
+        self._stdin = sys.stdin
+        self._stdout = sys.stdout.buffer
+        self._stderr = sys.stderr.buffer
```

Comment:
What do you think about getting rid of those variables and always using the global values from `sys`? I don't think that Python makes a clone of the streams; I think it's only a reference iiuc. It might make it more explicit that we're accessing some magical global value if you use `sys.stdin` rather than `self._stdin` at the call sites - more typing, but more explicit.